packages/experiments/src/components/PersonaCoin/CoinAlternativeForSmallestSize/CoinAlternativeForSmallestSize.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Icon } from 'office-ui-fabric-react';
+
+export default class PersonaCoinSize10 extends React.Component {
```

Comment:
The file name is a bit chunky. Is there a reason it isn't `PersonaCoinSize10.tsx`?

Reply:
No reason, good catch :D